Packs/Gurucul/Integrations/GuruculGRA/GuruculGRA.py

```diff
@@ -196,8 +196,11 @@ def main() -> None:
             proxy=proxy)
 
         if demisto.command() == 'test-module':
-            result = test_module_command(client)
-            return_results(result)
+            try:
+                result = test_module_command(client)
+                return_results(result)
+            except Exception:
```

Comment:
error handling already exist within the test module, rest of the errors are to be handled in the main 'except' block.